changes/5579.security

```diff
@@ -0,0 +1 @@
+Updated `sqlparse` to `0.5.0` to fix passing a heavily nested list to `sqlparse.parse()` leads to a Denial of Service due to RecursionError. This is not a direct dependency so it will not auto-update when upgrading Nautobot. Please be sure to update your local environment.
```

Comment:
I'd just reference https://github.com/advisories/GHSA-2m57-hf25-phgg instead of trying to explain the specific security issue here.